### PR TITLE
Fixed search bar bug for searchable collection view.

### PR DESCRIPTION
**Pull request by elland**

### CODE REVIEW

File: Sources/SearchableCollectionController.swift [hunk 1]
```diff
@@ -71,7 +73,7 @@ extension SearchableCollectionController: UICollectionViewDataSource {
 
 extension SearchableCollectionController: UICollectionViewDelegate {
     open func scrollViewDidScroll(_ scrollView: UIScrollView) {
-        guard self.isAnimatingSearchBar == false else { return }
+        guard self.isAnimatingSearchBar == false && !self.isActive else { return }
```
Comment (yuliaveres) [inline Sources/SearchableCollectionController.swift]: we should keep checks consistent **!** or **== false**

File: Sources/SearchableCollectionController.swift [hunk 2]
```diff
@@ -17,6 +17,8 @@ open class SearchableCollectionController: SweetCollectionController {
 
     fileprivate var isAnimatingSearchBar: Bool = false
 
+    fileprivate var isActive: Bool = false
```
Comment (yuliaveres) [inline Sources/SearchableCollectionController.swift]: why do u need :Bool here ?

File: Sources/SearchableCollectionController.swift (hunk 1 above)
Comment (roberthein) [reply Sources/SearchableCollectionController.swift]: Good one! 😉

**Review by roberthein — APPROVED** [review]
Looks ok.

**Comment by elland** [comment]
🆗 🚀

**Pull request merged**